src/main/charts/confluence/values.yaml

```diff
@@ -155,7 +155,7 @@ database:
     
 # REQUIRED - Volume configuration
 #
-# By default, the charts will configure the local-home and shared-home as ephemeral 
+# By default, the charts will configure the local-home, synchrony-home and shared-home as ephemeral
 # volumes i.e. 'emptyDir: {}'. This is fine for evaluation purposes but for production 
 # deployments this is not ideal and so local-home and shared-home should both be configured 
```

Comment:
Since we mentioned synchrony-home above, maybe here as well?

Reply:
Fixed